### PR TITLE
Metrics Webapp

### PR DESCRIPTION
**Pull request by Chimwemwe-127001**
# Metrics Webapp
> In this feature update:
## Technical requirements.
- [x] The project is a single page application (SPA) built with React and Redux.
- [x] The data retrieved from the API should be stored in the Redux store.
- [x] should filter the data that you retrieve from the API using a Filter stateless component.
- [x] Every page (the main page and the pages for each item) should have a unique route within the SPA.
- [x] The project should be deployed and accessible online (add a link to the online version of your app to your README file).

## Testing requirements
- [x] Create unit tests for pure functions in Redux code.
- [x] Create integration tests for your application using the React Testing Library

### CODE REVIEW

File: src/components/ItemDetails.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/* eslint-disable dot-notation */
+/* eslint-disable quotes */
```
Comment (bimbolabuari) [inline src/components/ItemDetails.js]: Let's remove all disabled Linters, Linting helps to achieve cleaner and more professional code  ✊ 

Kindly check for other occurrences 🙂

File: src/components/ListItems.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-nested-ternary */
```
Comment (bimbolabuari) [inline src/components/ListItems.js]: Let's remove all disabled Linters, Linting helps to achieve cleaner and more professional code ✊

Kindly check for other occurrences 🙂

File: src/components/ItemDetails.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            </td>
+            <td className="column">
+              {recovered}
+              {' '}
+              cases
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
```
Comment (bimbolabuari) [inline src/components/ItemDetails.js]: Let's stick to the required design template 👍 

Your details page design


>Screenshot (346)


Design template
>Screenshot (342)

File: src/components/ListItems.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                  <h4 className="companyName">{country.name}</h4>
+                </Link>
+                <div>
+                  <p className="casesConfirmedVal">{country.confirmedCases}</p>
+                  <p className="casesConfirmed">cases confirmed</p>
+                </div>
+              </div>
+            ))
+          )
+          : populate}
```
Comment (bimbolabuari) [inline src/components/ListItems.js:67]: Let's stick to the design guide as much as possible ✊ 

Your home page
>Screenshot (351)

Design Guideline home page
>Screenshot (347)

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hello @Chimwemwe-127001, 

You have done a good job so far 🚀  

## Highlights 🎉 

- The README contains a section that gives appropriate credit to Nelson Sakwa the author of the original design, as required by the Creative Commons License.
-  The project contains a `Home` page and `Detail` page.
- The Home page has list of categories that could be filtered using given parameters
- It's possible to navigate detail page by clicking the individual category item, there's a link from `Detail` page to `Home` page by clicking "Back" button.
- Deployed Web app
- Nice Video Presentation

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/components/ItemDetails.js (hunk 1 above)
Comment (Chimwemwe-127001) [reply src/components/ItemDetails.js]: Done

File: src/components/ListItems.js (hunk 2 above)
Comment (Chimwemwe-127001) [reply src/components/ListItems.js]: Done

File: src/components/ListItems.js (hunk 4 above)
Comment (Chimwemwe-127001) [reply src/components/ListItems.js:67]: Done

File: src/components/ItemDetails.js (hunk 3 above)
Comment (Chimwemwe-127001) [reply src/components/ItemDetails.js]: done

**Comment by Chimwemwe-127001** [comment]
- Thanks for the insightful review, @bimbolabuari. I have actioned all the requested changes. Hopefully, my PR gets approved this time.
- Happy coding :smiley: :computer:

**Comment by omar-labana** [comment]
Hi @Chimwemwe-127001, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**